.buildkite/build_vaxine.yml

```diff
@@ -1,5 +1,6 @@
 env:
-  DOCKER_REPO: europe-docker.pkg.dev/vaxine/vaxine-io
+  DOCKER_REPO: europe-docker.pkg.dev/vaxine/electric-sql
+  DOCKERHUB_REPO: electricsql
   IMAGE_NAME: vaxine
```

Comment:
Any benefit in having those envs in a separate file? I would just move all those variables to Makefile together with `export ELECTRIC_IMAGE_NAME="${DOCKERHUB_REPO}/${IMAGE_NAME}"` and would just run single step - `make buildkite-build-and-push`